core/playlistmanager.go

```diff
@@ -226,16 +233,30 @@ func NewBasicPlaylistManager(manifestID ManifestID,
 	if recordSession != nil {
 		bplm.jsonList = NewJSONPlaylist()
 		bplm.jsonListSync = &sync.Mutex{}
+		bplm.makeNewOverwriteQueue()
 	}
 	return bplm
 }
 
+func (mgr *BasicPlaylistManager) makeNewOverwriteQueue() {
+	if mgr.jsonListWriteQueue != nil {
+		mgr.jsonListWriteQueue.StopAfter(JsonPlaylistQuitTimeout)
+	}
+	mgr.jsonListWriteQueue = drivers.NewOverwriteQueue(mgr.recordSession, mgr.jsonList.name, "json playlist",
```

Comment:
Should the manifestID be used instead of "json playlist"?

Reply:
Added manifesID to the description